Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-gem 'kafo', '~> 6.0'
+gem 'kafo', '~> 6.4'
```

Comment:
https://github.com/theforeman/foreman-packaging/pull/6890 is the RPM PR. Is there a matching Debian PR?

Reply:
The Deb PR is -- https://github.com/theforeman/foreman-packaging/pull/6892